spec/openapi.yaml

```diff
@@ -963,6 +980,91 @@ components:
           type: string
           example: "/dev/tty"
 
+    StatusJob:
+      type: object
+      description: Telemetry info about current job, all values are optional
+      properties:
+        id:
+          type: integer
+          example: 42
+        progress:
+          type: number
+          example: 0.42
+        time_remaining:
+          type: integer
+          example: 520
+        time_printing:
+          type: integer
+          example: 526
+
+    StatusPrinter:
+      type: object
+      description: Telemetry info about printer, all values except state are optional
+      properties:
+        state:
+          type: string
+          enum: [ IDLE, BUSY, PRINTING, PAUSED, FINISHED, STOPPED, ERROR, ATTTENTION, READY ]
+        temp_nozzle:
+          type: number
+          example: 214.9
+        temp_bed:
+          type: number
+          example: 59.5
+        axis_x:
+          type: number
+          description: Available only when printer is not moving
+          example: 23.2
+        axis_y:
+          type: number
+          description: Available only when printer is not moving
+          example: 24.3
+        axis_z:
+          type: number
+          example: 0.5
+        flow:
+          type: integer
+          example: 95
+        speed:
+          type: integer
+          example: 100
+        fan_hotend:
+          type: integer
+          example: 420
+        fan_print:
+          type: integer
+          example: 420
+        connect_status:
```

Comment:
What about serial port status?

Reply:
As we spoke before, I'll add section printer_status, which includes the serial port status.